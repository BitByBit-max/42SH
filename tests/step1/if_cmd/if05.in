if echo hi; echo how are you; echo fine; then echo cool;fi
