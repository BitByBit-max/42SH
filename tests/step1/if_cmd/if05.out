hi
how are you
fine
cool
